src/Resolver/OrganizationIdResolver.php

```diff
@@ -22,7 +22,7 @@ public function __construct(ShippingGatewayRepositoryInterface $shippingGatewayR
         $this->shippingGatewayRepository = $shippingGatewayRepository;
     }
 
-    public function getOrganizationId(): string
+    public function getOrganizationId(): ?string
```

Comment:
I don't understand why you changed the type to be nullable. As I saw below, you check only empty string in the condition.